Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
+    money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.7)
+    rack (2.0.8)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.2.0)
-      loofah (~> 2.2, >= 2.2.2)
-    railties (6.0.0)
-      actionpack (= 6.0.0)
-      activesupport (= 6.0.0)
+    rails-html-sanitizer (1.3.0)
```

Comment:
### [rails-html-sanitizer](https://github.com/rails/rails-html-sanitizer)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.2.0 → 1.3.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/rails/rails-html-sanitizer/compare/v1.2.0...v1.3.0).

These are the individual commits:
- (43a87f5) [Match Loofah's API changes.](https://github.com/rails/rails-html-sanitizer/commit/43a87f538af42dd11cd72fddacbba2519d1e5c78)
- (845da04) [Add project metadata to the gemspec](https://github.com/rails/rails-html-sanitizer/commit/845da04cad37885aadc4495c8e7e476de54aa803)
- (65b9f88) [Merge pull request #102 from orien/gem-metadata](https://github.com/rails/rails-html-sanitizer/commit/65b9f880f23abe9c7f2719da58bdf613f5f5c307)
- (51dc564) [v1.3.0](https://github.com/rails/rails-html-sanitizer/commit/51dc564c6509201070f72456bb2c13f87bb373d6)


</details>